driver-core/src/main/com/mongodb/event/ConnectionRemovedEvent.java

```diff
@@ -23,7 +23,9 @@
  * An event for removing a connection from the pool.
  *
  * @since 3.5
+ * @deprecated Prefer {@link ConnectionClosedEvent}
  */
+@Deprecated
```

Comment:
Same as the ConnectionAdded comment - this looks like its not used at all.

Reply:
I agree with Jeff. Since this is a public event, we can't remove this until version 5 of the driver.